src/diffusers/models/unet_2d_blocks.py

```diff
@@ -35,7 +38,7 @@ def get_down_block(
     add_downsample,
     resnet_eps,
     resnet_act_fn,
-    attn_num_head_channels,
+    attn_num_heads=None,
```

Comment:
Why not use `num_attention_heads`? It's consistent with the rest of the changes, no?

Reply:
agree!